docs/manage/idp/azure-ad.md

```diff
@@ -25,26 +25,21 @@ infra providers add azure \
 3. Click **New registration**
 4. Register the application:
    - For **Name** write **Infra**
-   - For **Redirect URI** select `Web` and add `http://localhost:8301`
+   - For **Redirect URI** select `Web` and add `https://<INFRA_SERVER_HOST>/login/callback`
```

Comment:
Do we explain what `INFRA_SERVER_HOST` is anywhere in these docs? Might be a good follow up to say something about it (ex: generally `<org>.infrahq.com`).